crates/sui-gateway/src/unit_tests/rpc_server_tests.rs

```diff
@@ -24,7 +24,7 @@ use test_utils::network::start_rpc_test_network;
 
 #[tokio::test]
 async fn test_get_objects() -> Result<(), anyhow::Error> {
-    let test_network = start_rpc_test_network(None).await?;
+    let test_network = start_rpc_test_network(None, None).await?;
```

Comment:
would prefer if we didn't add more optional params - I agree with what @bmwill said about not using this method directly in your tests, and using the builder instead. Then, instead of adding optional params, add a (private) function `start_rpc_test_network_with_fullnode_count()` or whatever - `start_rpc_test_network` can then call it with the default value.